tests/tools/onert_train/src/onert_train.cc

```diff
@@ -138,11 +138,16 @@ int main(const int argc, char **argv)
     NNPR_ENSURE_STATUS(nnfw_train_get_traininfo(session, &tri));
 
     // overwrite training information using the arguments
-    tri.batch_size = args.getBatchSize();
-    tri.learning_rate = args.getLearningRate();
-    tri.loss_info.loss = args.getLossType();
-    tri.loss_info.reduction_type = args.getLossReductionType();
-    tri.opt = args.getOptimizerType();
+    if (tri.batch_size == 0)
+    {
+      // If args doens't have a batch size, set batch size 1
+      tri.batch_size = args.getBatchSize().value_or(1);
```

Comment:
if both the model file and cmd-line options doesn't give a batch size, 
Still make it run somehow, by setting batch_size as 1.